spec/jobs/psql_stats_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe PsqlStatsJob, type: :job do
+  describe '#perform' do
+    it 'returns true' do
+      result = PsqlStatsJob.new.perform
+
+      expect(result).to eq true
+    end
+
+    it 'logs psql table bloat metrics' do
+      expect(IdentityJobLogSubscriber.reports_logger).to receive(:info) do |str|
+        msg = JSON.parse(str, symbolize_names: true)
+        expect(msg[:name]).to eq('psql_bloat_statistics')
+        expect(msg[:table_data][:users][:tblname]).to eq('users')
```

Comment:
❤️

Reply:
thanks for the tips, that structure makes things much easier